int32 light_state
---
int32 code
string message